web/themes/joinup/prototype/sass/components/_featured.sass

```diff
@@ -140,6 +140,7 @@
   display: inline-block
   margin: 10px
   z-index: 99
+  vertical-align: middle
```

Comment:
When I used previously the dropdown button, there was a positioning issue. It is not needed anymore.